app/assets/javascripts/app/users/user.js

```diff
@@ -5,36 +5,40 @@
     .module('council.users')
     .controller('EditUserCtrl', EditUserController);
 
-  function EditUserController(User, userId, DS) {
+  function EditUserController($state, User, userId) {
```

Comment:
This controller is doing too many things.

* Fetching the user data
* Managing the form's submit
* Managing form validations
* Redirecting to some place in the success scenario

How would you clean this?